notary-server/config/config.yaml

```diff
@@ -5,13 +5,19 @@ server:
 
 notarization:
   max-transcript-size: 16384
+  session-ttl-seconds: 300
 
 tls-signature:
   private-key-pem-path: "./fixture/tls/notary.key"
   certificate-pem-path: "./fixture/tls/notary.crt"
 
 notary-signature:
   private-key-pem-path: "./fixture/notary/notary.key"
+  public-key-pem-path: "./fixture/notary/notary.pub"
 
 tracing:
   default-level: DEBUG
+
+authorization:
+  enabled: false
+  whitelist-csv-path: "./fixture/auth/whitelist.csv"
```

Comment:
For now, whenever we want to add/remove entry from the file, we need to restart the server to reload the whitelist file — not sure how else we can refresh it as this file is only loaded during startup to reduce latency — should definitely use a better solution in the future e.g. persistent db to allow hot reload

Reply:
nice will keep in mind!